src/gnssro/gnss_tec_ground_based.py

```diff
@@ -0,0 +1,468 @@
+#!/usr/bin/env python3
+
+#
+# (C) Copyright 2019-2024 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+#
+# decode TENET formatted ground-based GNSS Total Electron Content (TEC)
+#
+
+import sys
+import os
+import time
+from datetime import datetime, timezone
+import numpy as np
+import logging
+
+# These modules need the path to lib-python modules
+import pyiodaconv.ioda_conv_engines as iconv
+from pyiodaconv.orddicts import DefaultOrderedDict
+from pyiodaconv.def_jedi_utils import iso8601_string, epoch
+
+os.environ["TZ"] = "UTC"
+
+# these are the unique values in the raw input file
+varDict = {
+    'totalElectronContent': ['totalElectronContent', "integer", 'TECU'],
+}
+
+# these are the MetaData common to each input
+locationKeyList = [
+    ('latitude', 'float', 'degrees_north'),
+    ('longitude', 'float', 'degrees_east'),
+    ('pseudoRandomNoiseCode', 'integer', 'GNSS transmitter pseudoRandomNoise PRN code'),
+    ('elevationAngleGNSS', 'float', 'GNSS transmitter satellite elevation angle in degrees'),
+    ('azimuthAngle', 'float', 'aziumuth angle viewing GNSS transmitter in degrees west'),
```

Comment:
@BenjaminRuston regarding the new names, is this different from `beamAzimuthAngle` or `sensorAzimuthAngle` ?

Reply:
great catch @haydenlj we should pick one of those... can you help decide which is more appropriate?

it is a sensor, beam also works though....  would think this is being used for RADAR?

Reply:
I guess beam infers the observation location as the point of reference and here the beam originates at the transmitter.. so maybe `sensor` should be selected in this case as it's looking towards a source

Reply:
Yes that's the way I was thinking about it too, sounds good to me